### ISSUE TITLE
How to reproduce the  results

### ISSUE THREAD

**Issue by xizaoqu** [ML-GSAI/SDE-Drag#2]
Hi, thanks for your great work.
When I tried the demo, the result was very weird, any suggestions?
![image](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/01ef261f-7bb1-471e-ac85-e75140befb93)

**Comment by nieshenx** [comment]
Thank you for your interest and feedback！

Firstly, I try to reproduce your result. I didn't modify any parameters in the SDE-Drag GUI, the result is as follows:
![0](https://github.com/ML-GSAI/SDE-Drag/assets/68948796/8f3ea276-2063-4035-998d-7637851213ee)
Even though there may be tiny differences in the mask and points we provide in the GUI, my result is very different from yours. I guess there may be two reasons for your very blurry result: 1. An inappropriate prompt will lead to poor results.; 2. Although stable diffusion (including SDE-Drag) supports images of different resolutions, we know that stable diffusion performs best at a resolution of 512*512, so when the resolution is too large or too small, it may cause A bad result.

Secondly, as I mentioned in README, _fine-tuning LoRA is suggested if dragging changes the main subject, like a cat turning into another cat_.  So I tried to drag using LoRA, the result is as follows, which is slightly better:
![1](https://github.com/ML-GSAI/SDE-Drag/assets/68948796/35bd2ed4-ee63-43be-9851-a60e523ce1dc)

Thirdly, as mentioned in Sec. 5.4 of our paper, a higher _t_0_ improves image fidelity. So I increase the  parameter _t_0_ from the default 0.6 to 0.7, and get a slightly better result:
![t_0=0 7](https://github.com/ML-GSAI/SDE-Drag/assets/68948796/71b2a56f-f221-4d32-b848-c0ca8705176f)

Except for the content mentioned above, I am not adjusting any other parameters.  If you want better results, you can try adjusting other parameters, such as the random seed. Our algorithm is an SDE-based algorithm (randomness is introduced during editing), and different random seeds may lead to different results. 

Finally, thank you for your question. Looking forward to your reply so that we can further improve our methods.

**Comment by xizaoqu** [comment]
Thanks for your detailed reply! I tried again and it worked. Former weird results might be because of my environmental corruption.
![out_image_final](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/dd26e3e7-bcc9-4c36-870c-a97d3883621f)

**Comment by xizaoqu** [comment]
I also tried other cases, but the results were not so satisfying. Do you have some insights about it? (I didn't change the setting)
![image](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/562746c6-79a9-4569-9fde-e5dcc5177b63)
![image](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/e12b3ff5-f8ff-4b85-ab07-d043a4ce2a5b)

origin photos
![a](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/83a76b80-bc8a-4c01-982c-e428eac68bd3)
![dog6_02](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/8a41d5bf-c664-49b4-a952-b792252f0df9)

**Comment by nieshenx** [comment]
Thank you very much for your reply!
In this case, the object was dragged but changed (for example, the dog's appearance changed), so using LoRA will achieve better results. The button for using LoRA is under the leftmost image. In our interface, the default is to cancel loRA finetuning.

**Comment by xizaoqu** [comment]
Thanks for your reply!
I tried Lora, the photo of the log is much better now, but the man did not change a lot.
![20231119-134219](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/6c9faa04-1347-4762-9d4b-65374c210ba8)
![20231119-134223](https://github.com/ML-GSAI/SDE-Drag/assets/45515569/c04c3060-3458-45fe-9a75-e498d1c42849)

**Comment by nieshenx** [comment]
Sorry for not noticing your message in time.
I tried your demo and found that this is not a simple problem for our algorithm. I reduced the drag distance, and then in order to prevent overfitting (the edited image is consistent with the original image), I adjusted the "Min LoRA scale" to 0.2, and then did not adjust any other parameters, and finally got this result. Note that due to the front-end display, part of the content in the image editing in the UI interface is covered.
![微信截图_20231123235439](https://github.com/ML-GSAI/SDE-Drag/assets/68948796/c0f16d8d-3ec8-4654-8141-0e9dd460c450)

**Comment by xizaoqu** [comment]
> Sorry for not noticing your message in time. I tried your demo and found that this is not a simple problem for our algorithm. I reduced the drag distance, and then in order to prevent overfitting (the edited image is consistent with the original image), I adjusted the "Min LoRA scale" to 0.2, and then did not adjust any other parameters, and finally got this result. Note that due to the front-end display, part of the content in the image editing in the UI interface is covered. ![微信截图_20231123235439](https://private-user-images.githubusercontent.com/68948796/285791863-c0f16d8d-3ec8-4654-8141-0e9dd460c450.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMzYzMDYsIm5iZiI6MTcwMTMzNjAwNiwicGF0aCI6Ii82ODk0ODc5Ni8yODU3OTE4NjMtYzBmMTZkOGQtM2VjOC00NjU0LTgxNDEtMGU5ZGQ0NjBjNDUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDA5MjAwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhNTM3YzFiZDAyZDJmNjI3ZTFiYjYxNjI2ZWNhYjcwMWEwNGRlYjc5ODExMzFkNWMzN2EyYmIwZjA1ODYzMWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vN4YSLe8jfXjHVUvQE4GjZtpfiiNctTgIi9toygLIxk)

Thanks for your reply and insights~
